ir/value.h

```diff
@@ -74,6 +74,14 @@ class VoidValue final : public Value {
 };
 
 
+class NullPointerValue final : public Value {
```

Comment:
Why do we need a NullPointerValue and a NullPointerConst?
Isn't the const sufficient?

Reply:
Yeah, NullPointerValue was enough. I removed NullPointerConst